README.md

```diff
@@ -1,2 +1,16 @@
-# project-templates
-Open Liberty Maven and Gradle project templates
+![](https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png)
+
+This repository contains starter templates for Open Liberty.
+	
+## Templates
+ - gradle
+    - This template contains the core files needed for Open Liberty and Gradle to run.
+ - maven
+    - This template contains the core files needed for Open Liberty and Maven to run.
+  - simple-rest-app	
+    - This template contains both Gradle and Maven core files to run a simple greeting REST endpoint.
+    - Once started, the endpoint can be reached at http://localhost:9080/api/resource/greeting
+    
+## Liberty Starter
+The [Liberty Starter](http://start.openliberty.io) page can be used to generate maven or gradle projects.
```

Comment:
Is this live?  If not, we should remove this section.

Reply:
Removed!